src/main/java/com/blackducksoftware/integration/hub/api/component/version/ComplexLicenseItem.java

```diff
@@ -64,4 +67,8 @@ public ComplexLicenseType getType() {
         return type;
     }
 
```

Comment:
We shouldn't need to override toString, as the behavior you are after is in HubItem